frontend/constants.ts

```diff
@@ -18,3 +18,5 @@ export const SPRING_2021_TRANSITION_MESSAGE_TOKEN =
     "__spring_2021_transition_message_shown";
 export const ANALYTICS_SURVEY_SHOWN_SPRING_2021_END_TOKEN =
     "__analytics_survey_sp_2021_end_shown";
```

Comment:
Can I delete the above 3 constants?

Reply:
I would lean towards not deleting them, just so we don't accidentally choose keys that conflict with these in the future. As these grow larger we might want to move them into a separate file of have a better API to create/access new keys.